qbraid_qir/qiskit/visitor.py

```
@@ -0,0 +1,289 @@
+
+        Args:
+            module: The QiskitModule to visit.
+        """
+        logger.debug(
+            "Visiting Qiskit module '%s' (%d qubits, %d clbits)",
+            module.name,
+            module.num_qubits,
+            module.num_clbits,
+        )
+        if module.module is None:
+            raise ValueError("QiskitModule must have a PyQIR module set before visiting.")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)